File: app/src/main/java/com/tutozz/blespam/Spammer.java
package com.tutozz.blespam;

import org.jetbrains.annotations.NotNull;

public interface Spammer {
    @NotNull

    boolean isSpamming();
    void start();
    void stop();
    void setBlinkRunnable(Runnable blinkRunnable);
    Runnable getBlinkRunnable();
}

File: app/src/main/java/com/tutozz/blespam/Spammer.kt
package com.tutozz.blespam

import android.bluetooth.le.AdvertiseCallback

interface Spammer {
    fun start()
    fun stop()
    fun isSpamming(): Boolean

    fun setBlinkRunnable(blinkRunnable: Runnable?)
    fun getBlinkRunnable(): Runnable?
}
